docs/src/ToricVarieties/ToricDivisors.md

```diff
@@ -62,3 +62,11 @@ coefficients(td::ToricDivisor)
 polyhedron(td::ToricDivisor)
 toric_variety(td::ToricDivisor)
 ```
+
+## Special divisors
+
+```@docs
+TrivialDivisor(v::AbstractNormalToricVariety)
+AnticanonicalDivisor(v::AbstractNormalToricVariety)
+CanonicalDivisor(v::AbstractNormalToricVariety)
```

Comment:
Probably `snake_case` as above.

I see that `*DivisorClass` is not just `divisor_class(*Divisor)`, but would it be computationally much harder to just go the second route? Because then we could leave out all the `*DivisorClass` and have less code to maintain.

Reply:
Agreed. `snake_case` it is. Also, updating this such that these functions indeed just execute `DivisorClass(...)` on the corresponding divisor.